js/core/utils/style.js

```diff
@@ -111,8 +111,26 @@ const setHeight = function(elements, value) {
     setDimensionProperty(elements, 'height', value);
 };
 
-const setStyle = function(element, value) {
-    element.style.cssText = value;
+const setStyle = function(element, styleString, resetStyle = true) {
+    if(!domAdapter.isNode(element.style)) {
```

Comment:
transferred comment into style.js
but i consider the ticket id quite helpful in this comment because i was able to understand the problem better with ticket information